README.md

```diff
@@ -1,34 +1,75 @@
-[![Build Status](https://travis-ci.org/ServiceInnovationLab/fsd-spike.svg?branch=master)](https://travis-ci.org/ServiceInnovationLab/fsd-spike)
-[![Waffle.io - Columns and their card count](https://badge.waffle.io/ServiceInnovationLab/fsd-spike.png?columns=all)](https://waffle.io/ServiceInnovationLab/fsd-spike?utm_source=badge)
+# Family Services Directory Searchinator
+[![Build Status](https://travis-ci.org/ServiceInnovationLab/FamilyServiceDirectorySearchinator.svg?branch=master)](https://travis-ci.org/ServiceInnovationLab/FamilyServiceDirectorySearchinator)
+[![Waffle.io - Columns and their card count](https://badge.waffle.io/ServiceInnovationLab/FamilyServiceDirectorySearchinator.png?columns=all)](https://waffle.io/ServiceInnovationLab/FamilyServiceDirectorySearchinator?utm_source=badge)
 
-This runs [GH-Pages](https://serviceinnovationlab.github.io/fsd-spike/)
+This runs [on github pages](https://serviceinnovationlab.github.io/FamilyServiceDirectorySearchinator/)
```

Comment:
Yeah this is confusing, but I think it should reflect the real link for now, if you follow that link it shows an old codebase

Reply:
Fixed